packages/sprotty-protocol/src/model.ts

```diff
@@ -45,31 +45,22 @@ export interface ViewportRootElement extends SModelRoot, Partial<Viewport>, Part
 /**
  * Root element for graph-like models.
  */
-export interface SGraph extends ViewportRootElement {
+export interface SGraph extends ViewportRootElement, Partial<LayoutableChild> {
     children: SModelElement[]
-    layoutOptions?: ModelLayoutOptions
 
     /** @deprecated Use `position` and `size` instead. */
     bounds?: Bounds
 }
 
-/**
- * Options to control the "micro layout" of a model element, i.e. the arrangement of its content
- * using simple algorithms such as horizontal or vertical box layout.
- */
-export type ModelLayoutOptions = { [key: string]: string | number | boolean };
-
-export interface SShapeElement extends SModelElement, Partial<BoundsAware> {
-    layoutOptions?: ModelLayoutOptions
+export interface SShapeElement extends SModelElement, Partial<LayoutableChild> {
```

Comment:
`SShapeElement` is not `BoundsAware` anymore. Was that intended?
`SNode`, `SPort`, and `SLabel` don't have `position` and `size` anymore, or do I overlook something?

Reply:
Ah, it is inherited via `LayoutableChild`? The name is a bit misleading, isn't it?

Reply:
Do you have an alternative proposal?